docs/src/main/asciidoc/README.adoc

```diff
@@ -49,6 +49,11 @@ The UI should look like the screenshot above.
 
 include::https://raw.githubusercontent.com/spring-cloud/spring-cloud-build/master/docs/src/main/asciidoc/building.adoc[]
 
+IMPORTANT: There are 2 different versions of JDK used in Spring Cloud Sleuth. JDK7 is used for main sources and
```

Comment:
is this true? I think only one JDK is used. The comment is more about language level than JDK, right?